packages/dev-frontend/src/components/Dashboard/PriceCard.tsx

```diff
@@ -73,8 +73,9 @@ export const RedemptionRateCard = ({ variant = "mainCards" }: SystemStatsProps):
 };
 
 function annualizeInterestRate(redemptionRate: Decimal): string {

```

Comment:
I think we should keep the APY but add a new card for the EOT yield. That way it's clear that they're different (for someone who's not paying attention) and we will want to use the APY after the tournament is over.

Reply:
We should also change the title to be something like `LED Yield During Tournament` instead of `LED APY`